include/libdnf/conf/option.hpp

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (C) 2018-2020 Red Hat, Inc.
+Copyright (C) 2018-2021 Red Hat, Inc.
```

Comment:
What about to add into the commit message information why there is no method unlock. I suggest that the reason is missing plan for sack reset functionality therefore we do not need it.